modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,236 @@
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | Name for this configuration. Connectors reference the configuration with this name | None | *Yes*
+| Api Definition | String |  API definition | None | *Yes*
+| Kafka Configs | Array of <<KafkaConfig>> | None | No |  
+| Anypoint MQ Configs | Array of <<AnypointMqConfig>> | None | No |  
+| Name | String | Identifier of this element used to reference it in other components | None | *Yes*
+|===
+
+=== Supported Message Brokers
+
+[%header%autowidth.spread]
+|===
+| Feature | Description | Supported Protocols | Configuration Details 
+| Anypoint MQ Message Broker | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+| Kafka Message Broker | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+|===
+
+==== Associated Operations
+* <<Publish>>  
+
+==== Associated Sources
+* <<MessageListener>>  
+
+== Operations
+
+[[Publish]]
+== Publish
+`<apikit-asyncapi:publish>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Content | Binary | None | #[payload] | No  
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Server Name | String | Server name | None | *Yes* 
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No
+| Error Mappings | Array of <<ErrorMapping>> | Set of error mappings | None | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | PublishResponse
+| *Attributes Type* | Any
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+=== Throws
+
+* APIKIT-ASYNCAPI:ASYNCAPI_KIT_INTERNAL_SERVER_ERROR  
+* APIKIT-ASYNCAPI:INTERNAL_SERVER_ERROR
+* APIKIT-ASYNCAPI:PUBLISHING_ERROR
+* APIKIT-ASYNCAPI:VALIDATION_ERROR
+
+== Sources
+
+[[MessageListener]]
+== Message Listener
+`<apikit-asyncapi:message-listener>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Servers | Array of String | List of servers to listen to | None | No 
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
```

Comment:
I think it's weird to use outputs this way.

Reply:
Let's try with "generates".